dotCMS/src/main/java/com/dotmarketing/init/DotInitScheduler.java

```diff
@@ -635,6 +638,33 @@ private static void scheduleJob(final JobBuilder jobBuilder) {
 		}
 	}
 
+	private static void addStartEndScheduledExperimentsJob(final Scheduler scheduler) {
+		try {
+			final String jobName      = "EndFinalizedExperimentsJob";
+			final String triggerName  = "trigger31";

```

Comment:
AccessTokenRenewJob has the trigger name `trigger31` and group name `group31` reserved.

Reply:
Updated to 32